### PR TITLE
[Wait for #2567] [ Test ] Mixed Precision Test Case 

### PR DESCRIPTION
**Pull request by jijoongmoon**
## In this PR
This PR includes the mixed precision test case.

. Input - FC - MSE
 : "batch_size=2", "model_tensor_type=FP16-FP16", "loss_scale=128"

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/layers/input_layer.cpp [hunk 1]
```diff
@@ -70,7 +69,21 @@ void InputLayer::finalize(InitLayerContext &context) {
...
+  for (auto &d : output_dims) {
+    d.setDataType(context.getActivationDataType());
+  }
+
   context.setOutputDimensions(output_dims);
+
+  is_inplace = true;
+
+  /**
+   * @note Input Layer assuems that the FP32 IN Tensor always. Therefore, if the
```
Comment (DonghakPark) [inline nntrainer/layers/input_layer.cpp:81]: typo ```assume```  ?

**Review by DonghakPark — APPROVED** [review]

File: nntrainer/tensor/tensor.h [hunk 2]
```diff
@@ -1680,6 +1680,13 @@ class Tensor {
    */
   Tensor clone() const;
 
+  /**
+   * @brief     Convient wrapper for inplace copy of @a this.
```
Comment (EunjuYang) [inline nntrainer/tensor/tensor.h:1684]:
```suggestion
   * @brief     Convenient wrapper for inplace copy of @a this.
```
Is it typo? Do you mean convenient ?

File: nntrainer/tensor/weight.h [hunk 3]
```diff
@@ -114,6 +114,7 @@ class Weight : public Var_Grad {
    *
    * @param v Already created variable object
    * @param g Already created gradient object
+   * @param v32 Already created gradient object
```
Comment (EunjuYang) [inline nntrainer/tensor/weight.h:117]:
```suggestion
   * @param v32 Already created variable32 object
```

**Comment by jijoongmoon** [comment]
closed by  #2663

**Pull request closed without merging**